### PR TITLE
Update enroll-device-linux.md

### PR DESCRIPTION
**Pull request by msbemba**
added a note that ubuntu on wsl2 is not supported.

reference - https://learn.microsoft.com/en-us/windows/wsl/enterprise

fixes#https://github.com/MicrosoftDocs/memdocs/issues/3554

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/user-help/enroll-device-linux.md [hunk 1]
```diff
@@ -64,5 +64,8 @@ Follow these steps to register your personal Linux device on your organization's
    2.  An on-screen confirmation appears when your device is enrolled and ready-to-use for work. You can begin using your device for work right away. 
    3. Sign in to Microsoft Edge with your work or school account to access your org's internal websites.   
 
+> [!NOTE]
+> Ubuntu on WSL2 is not a supported scenario and the requests is in on our backlog and we are investigating ways to add them.
```
Comment (JohanFreelancer9) [inline memdocs/intune/user-help/enroll-device-linux.md:68]:
```suggestion
> Currently, Ubuntu on WSL2 is not supported but we are investigating ways to add support for it in the future..
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by rovinbhandari — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please signoff this PR. Thanks

**Comment by lenewsad** [comment]
Hi @msbemba @yogkumgit, thanks for the ping. Waiting on PM to approve change.

**Comment by lenewsad** [comment]
Hi @msbemba and @yogkumgit, thank you for working through this PR. Confirmed w/PM that it's not currently supported but no word on plans for future support, so I removed that part.

**Comment by lenewsad** [comment]
#sign-off

**Review by lenewsad — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**